analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -287,7 +287,8 @@ class Pip(
         // Amend information from "setup.py" with that from "requirements.txt".
         val projectName = when (Pair(setupName.isNotEmpty(), requirementsName.isNotEmpty())) {
             Pair(true, false) -> setupName
-            Pair(false, true) -> requirementsName
+            // As requirementsName cannot be made very unique use the definition file path as projectName:
+            Pair(false, true) -> definitionFile.relativeToOrNull(analyzerRoot)!!.toString()
```

Comment:
Why do you use the "OrNull" variant here, but then cast it to non-null via "!!"?

Reply:
If you'd simply use `toRelativeString()` you could even spare the `toString()` in the end.

Reply:
good point, wasn't aware of that one.